src/modules/cncjs/filesystem/root/etc/haproxy/haproxy.cfg

```diff
@@ -45,7 +45,7 @@ backend octoprint
         errorfile 503 /etc/haproxy/errors/503-no-octoprint.http
 
 backend webcam
-        reqrep ^([^\ :]*)\ /webcam[/]?(.*)     \1\ /\2
+        http-request replace-uri ^([^\ :]*)\ /webcam[/]?(.*)     \1\ /\2
```

Comment:
This needs a review as this is the bit I am most unsure about and have not been able to test at all

Reply:
This did not work. I grabbed this line originally from the octopi settings. Guysoft has two versions of this file now (one for v1 and one for v2 of haproxy): 
https://github.com/guysoft/OctoPi/tree/devel/src/modules/octopi/filesystem/root/etc/haproxy

Some of the other changes (like min to m) aren't present, which is suspicious... The octoprint regex looks a little more generic too (it might work better for things like the /api).

At any rate, I have an updated line for the webcam coming in.